README.md

```diff
@@ -52,12 +53,34 @@ Note that the auspice development mode uses the auspice splash page which won't
 This is because the nextstrain.org server (see above) uses the splash page of the static content, but the auspice dev server doesn't.
 The advantage of development mode is that the client will live update as you edit the customisations.
 
-
+---
 ## Server
-`./server.js` decides, based on the path, whether to serve auspice or the (pre-built) static documentation / splash pages.
-It also imports auspice-specific code from `./auspice/server` to process auspice requests for data (including data transforms).
+Auspice is a flexible tool created for visualising phylogenomic data that are not specific to any domain.
+Therefore, auspice installations do not come with any pre-packaged data.
+Auspice requires interaction with a server to send and receive data.
+The [auspice repo on GitHub](http://github.com/nextstrain/auspice) contains a NodeJS backend that serves the bundled auspice JavaScript file to the server and also provides web API endpoints (prefixed with '`/charon`).
+The auspice front-end (e.g. "client") is server-agnostic. Currently, however, the auspice backend (e.g. "server") is not implemented in an importable or flexible way that allows usage across servers.
+This means that nextstrain.org, auspice.us, and your local auspice installation all use different servers.
 
+[./server.js](server.js) decides, based on the path, whether to serve the bundled auspice JavaScript file (built from running `auspice build`), the (pre-built) static [splash & documentation pages](#Splash--documentation-pages), or the auspice-specific code from `./auspice/server` to process auspice requests for data from the `/charon` web API endpoints (including data transforms).
 
+---
+## Build nextstrain.org locally
```

Comment:
These are great! Can we separate out "Build nextstrain.org locally mirroring the deployed (live) website", which _literally_ runs these 3 steps, and "Build nextstrain.org locally using a custom version of auspice" which is only really useful for development purposes.